src/utilities/core/jsoncpp.i

```diff
@@ -0,0 +1,90 @@
+#ifndef JSONCPP_I
+#define JSONCPP_I
+
+%{
+  #include <json/value.h>
+%}
+
+#if defined SWIGPYTHON
+%fragment("JsonToDict","header", fragment="SWIG_FromCharPtrAndSize") {
+  inline PyObject* SWIG_From_JsonValue(const Json::Value& value) {
```

Comment:
I ended up moving the fragments defined in xxx/LanguageSpecific.i to a single jsoncpp.i file... Because I was getting a "multiple defined symbol SWIG_From_JsonValue" when linking the ruby libs into a single lib. I'm not 100% sure but really I think the only think that fixed it is `inline` here. I'm fine leaving it in jsoncpp.i though